src/style.css

```
@@ -0,0 +1,30 @@
+:root {
+  --height-box: 16px;
+  --width-box: 20px;
+}
+
+.container {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat;
+  background-position: 2px 2px;
+  height: var(--height-box);
+  width: var(--width-box);
```

```suggestion
  height: var(--star-size);
  width: var(--star-size);
```
and add margins to make it work like this